data/categories.txt

```diff
@@ -5041,39 +5041,269 @@ de:Leuchtturm
 shop-copyshop
 en:Copy Shop|Printer
 ru:Копировальный центр|печать|полиграфия
+ar:محل نسخ|طباعة
+cs:Kopírovací obchod|Tiskárny
+de:Kopierladen|Drucker
+es:Copistería|imprenta|poligrafía
+fi:Painotalo|Tulostaja
+fr:Boutique de photocopies|Imprimerie
+hu:Fénymásoló üzlet|Nyomtató
+id:Fotokopi|Printer
+it:Copisteria|Stampe
+ja:コピーショップ|印刷会社
+ko:복사 상점|프린터
+nb:Kopieringsbutikk|Trykkeri
+pl:Punkt ksero|Drukarnia
+pt:Copiadora|Impressões
+ro:Centru copiere|Imprimare
+sk:Kopírovacie služby|Tlačiarne
+sv:Kopieringsbutik|Tryckare
+th:ร้านถ่ายเอกสาร|เครื่องพิมพ์
+tr:Kopyalama Merkezi|Yazıcı
+uk:Копіювальний центр|друк|поліграфія
+vi:Cửa hàng Copy|Máy in
+zh-Hans:复印店|打印机
+zh-Hant:復印店|打印機
 
 shop-photo
 en:Photo Shop|frames|shop
 ru:Фототовары|магазин
```

Comment:
Крафт - это ипэшник фотограф, я же специально читал описание.

Место, где делают фотографии на паспорт - это что?

> On Mar 25, 2016, at 23:22, Ilya Zverev <notifications@github.com> wrote:
> 
> In data/categories.txt <https://github.com/mapsme/omim/pull/2545#discussion_r57466080>:
> 
> > +id:Fotokopi|Printer
> > +it:Copisteria|Stampe
> > +ja:コピーショップ|印刷会社
> > +ko:복사 상점|프린터
> > +nb:Kopieringsbutikk|Trykkeri
> > +pl:Punkt ksero|Drukarnia
> > +pt:Copiadora|Impressões
> > +ro:Centru copiere|Imprimare
> > +sk:Kopírovacie služby|Tlačiarne
> > +sv:Kopieringsbutik|Tryckare
> > +th:ร้านถ่ายเอกสาร|เครื่องพิมพ์
> > +tr:Kopyalama Merkezi|Yazıcı
> > +uk:Копіювальний центр|друк|поліграфія
> > +vi:Cửa hàng Copy|Máy in
> > +zh-Hans:复印店|打印机
> > +zh-Hant:復印店|打印機
> >  
> >  shop-photo
> >  en:Photo Shop|frames|shop
> >  ru:Фототовары|магазин
> Нет, фотоателье — это craft=photographer.
> http://wiki.openstreetmap.org/wiki/Tag:shop%3Dphoto <http://wiki.openstreetmap.org/wiki/Tag:shop%3Dphoto>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2545/files/dedc3e589cea064617477f3b3d7d368809428cff#r57466080>